low-level/files/opt/seagate/sspl/setup/sspl_config.py

```diff
@@ -27,186 +27,96 @@
-        self._script_dir = os.path.dirname(os.path.abspath(__file__))
-        # Load sspl and global configs
-        Conf.load(consts.SSPL_CONFIG_INDEX, consts.sspl_config_path)
-        global_config_url = Conf.get(
-            consts.SSPL_CONFIG_INDEX,
-            "SYSTEM_INFORMATION>global_config_copy_url")
-        Conf.load(consts.GLOBAL_CONFIG_INDEX, global_config_url)
-        self.product = Conf.get(consts.GLOBAL_CONFIG_INDEX, 'release>product')
+        """Initialize varibales for Config stage."""
         self.topic_already_exists = "ALREADY_EXISTS"
+        Conf.load(consts.SSPL_CONFIG_INDEX, consts.sspl_config_path)
 
     def validate(self):
-        """Validate config for supported role and product"""
-        # Validate role
-        self.role = Conf.get(consts.GLOBAL_CONFIG_INDEX, 'release>setup')
-        if not self.role:
-            raise SetupError(
-                errno.EINVAL,
-                "%s - validation failure. %s",
-                self.name,
-                "Role not found in global config copy.")
-        if self.role not in consts.setups:
-            raise SetupError(
-                errno.EINVAL,
-                "%s - validation failure. %s",
-                self.name,
-                "Role '%s' is not supported. Check Usage" % self.role)
-
-    def replace_expr(self, filename: str, key, new_str: str):
+        """Check for below requirement.
 
-        """The function helps to replace the expression provided as key
-            with new string in the file provided in filename
-            OR
-            It inserts the new string at given line as a key in the
-            provided file.
+        1. Validate input configs
+        2. Validate sspl conf is created
+        3. Message bus is accessible
         """
-
-        with open(filename, 'r+') as f:
-            lines = f.readlines()
-            if isinstance(key, str):
-                for i, line in enumerate(lines):
-                    if re.search(key, line):
-                        lines[i] = re.sub(key, new_str, lines[i])
-            elif isinstance(key, int):
-                if not re.search(new_str, lines[key]):
-                    lines.insert(key, new_str)
-            else:
-                raise SetupError(
-                    errno.EINVAL,
-                    "Key data type : '%s', should be string or int",
-                    type(key))
-            f.seek(0)
-            for line in lines:
-                f.write(line)
-
-    def config_sspl(self):
-        if (os.geteuid() != 0):
-            raise SetupError(
-                errno.EINVAL,
-                "Run this command with root privileges!!")
-
+        if os.geteuid() != 0:
+            raise SetupError(errno.EINVAL,
+                "Run this command with root privileges.")
+
+        # Validate input/provisioner configs
+        machine_id = Utility.get_machine_id()
+        cluster_id = Utility.get_config_value(consts.PRVSNR_CONFIG_INDEX,
```

Comment:
What is the use of this variable?

Reply:
Removed assignment. But validation purpose we still keep this line.
Done.